docker-compose.yml

```diff
@@ -25,6 +25,8 @@ services:
       BPA_DID_PREFIX: ${BPA_DID_PREFIX}
       BPA_BOOTSTRAP_UN: ${BPA_BOOTSTRAP_UN}
       BPA_BOOTSTRAP_PW: ${BPA_BOOTSTRAP_PW}
+      ACAPY_ADMIN_URL: http://aca-py:${AGENT_ADMIN_PORT}
+      ACAPY_AGENT_URL: http://aca-py:${AGENT_HTTP_PORT}
```

Comment:
Not needed as it is already defined in the JAVA_OPTS

Reply:
Removed